polaris.shopify.com/src/components/StatusBanner/StatusBanner.tsx

```diff
@@ -11,7 +11,10 @@ function StatusBanner({status, children}: Props) {
   return (
     status && (
       <div className={styles.StatusBanner} data-value={status.toLowerCase()}>
-        <h2>{uppercaseFirst(status)}</h2>
+        <h2>
+          {uppercaseFirst(status)}
+          {status === 'Internal' ? ' only' : null}
```

Comment:
Is this necessary? I can't find any instances of `frontmatter.status` being set to `Internal`. 

<img width="460" alt="12-05-w47u5-w2tqa" src="https://github.com/Shopify/polaris/assets/1360097/ec194c75-c917-4dc3-a9b8-6b89a7d0d740">